components/collection/CollectionInfo.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div
+    class="is-flex is-justify-content-space-between mobile-flex-direction-column gap">
+    <div class="is-flex is-flex-direction-column is-flex-grow-1 max-width">
+      <div v-if="collectionInfo?.currentOwner" class="is-flex mb-2">
+        <div class="mr-2">{{ $t('activity.creator') }}</div>
+        <IdentityIndex
+          :address="address"
+          hide-identity-popover
+          :show-clipboard="false"
+          class="has-text-link has-text-weight-normal" />
```

Comment:
```suggestion
        <nuxt-link
          :to="`/${urlPrefix}/u/${address}`"
          class="has-text-link">
          <IdentityIndex ref="identity" :address="address" />
        </nuxt-link>
```
Something like this will work? Instead of your weird css fix

Reply:
Thanks!
That works